cwd/app.js

```diff
@@ -49,20 +51,23 @@
           var index = arr.indexOf(document.getElementById(this.id));
```

Comment:
I don't know how I missed this earlier a couple weeks ago, but could you use a more descriptive name than `arr`?  (it is created two lines above).  I guess it is an array of view buttons, but the title would not indicate that.

Also the slice on line 48 hardcodes this to only handle up to 3 view buttons.  Is that truly necessary?

Also, we should always try to avoid `var` when possible.  See if we can use `let` or `const`.
https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75
My rule is I want all my code to be written in a way that uses `const` only.  Sometimes it gets hard.  It's my ongoing challenge for your JavaScript code.

Reply:
All good comments, I found a better way than the hard-coding, so now more than 3 buttons should be handled, as long as naming remains consistent. Note that this would just be much easier if buttons were in their own class.

Reply:
I am very content with reducing the abstraction present across this application - if you'd like to either make a card to investigate further or just a personal note, I am happy to discuss it more later.